### PR TITLE
Mood

### PR DESCRIPTION
**Pull request by FomasTreeman**

### CODE REVIEW

**Comment by FomasTreeman** [comment]
Resolves #7

File: src/app/mood/page.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import Image from "next/image";
+import Link from "next/link";
+import emo1 from "../../public/images/emo1.png";
+import emo2 from "../../public/images/emo2.png";
+import emo3 from "../../public/images/emo3.png";
+import emo4 from "../../public/images/emo4.png";
+import styles from "./page.module.css";
+import { josefinSans } from "@/utils/fonts";
+import Exit from "@/components/Exit";
+import emo5 from "@/public/images/emo5.png";
```
Comment (cameochoquer) [inline src/app/mood/page.js:13]: next edit we should move this so they're all in line? or maybe we make this an array if possible?

File: src/public/images/emo5.png
Comment (cameochoquer) [inline src/public/images/emo5.png:1]: Should we change this so it's the same px quality as the others?

**Review by cameochoquer — APPROVED** [review]
Added comments on lines where I had questions/comments but most looks good.

**Review by zakkariyaa — APPROVED** [review]

**Pull request merged**